js-exercises/on-change/onChange.test.js

```diff
@@ -0,0 +1,72 @@
+import { onChange } from './onChange';
+
+describe('onChange', () => {
+//   test('main', () => {
+//     const fixture = {
+//       foo: false,
+//       bar: {
+//         a: {
+//           b: 0,
+//           c: [1, 2],
+//         },
+//       },
+//     };
+
+//     let callCount = 0;
+
+//     const object = onChange(fixture, () => {
+//       callCount += 1;
+//     });
+
+//     object.foo = true;
+//     expect(callCount).toBe(1);
+
+//     Object.defineProperty(object, 'newProp', {
+//       value: '🦄',
+//     });
+//     expect(callCount).toBe(2);
+
+//     Object.assign(object, {
+//       foo: false,
+//     });
+//     expect(callCount).toBe(3);
+
+//     delete object.foo;
+//     expect(callCount).toBe(4);
+
+//     object.bar.a.b = 1;
+//     expect(object.bar.a.b).toBe(1);
+//     expect(callCount).toBe(5);
+
+//     object.bar.a.c[2] = 5;
+//     expect(object.bar.a.c[2]).toBe(5);
+//     expect(callCount).toBe(6);
+//   });
```

Comment:
I have a question,
Why these test cases are commented?

Reply:
That's my bad actually. I commented them out to test a specific test case and forgot to uncomment them.
I'll fix it